mailchimp3/mailchimpclient.py

```diff
@@ -46,9 +46,15 @@ def __init__(self, mc_user, mc_secret, enabled=True):
         """
         super(MailChimpClient, self).__init__()
         self.enabled = enabled
-        self.auth = HTTPBasicAuth(mc_user, mc_secret)
-        datacenter = mc_secret.split('-').pop()
-        self.base_url = 'https://{}.api.mailchimp.com/3.0/'.format(datacenter)
+        if access_token:
```

Comment:
The how of receiving an OAuth access token is not provided in this pull request. Please update the README files with documentation in the Initialization section indicating the process for connecting the wrapper to mailchimp assuming that the only information a user will have is their standard credentials for accessing an existing MailChimp account. Add another level of headers under Initialization to indicate the Username/Secret Key information and the OAuth Token information. The information should be written so that we can enable OAuth and hook up a client without reading any of the developer portal documentation.